build.psm1

```diff
@@ -1442,6 +1442,15 @@ function Start-PSBootstrap {
 
         # Install Windows dependencies if `-Package` or `-BuildWindowsNative` is specified
         if ($Environment.IsWindows) {
+            ## need RCEdit to modify the binaries embedded resources
+            if (-not (Test-Path "~/.rcedit/rcedit-x64.exe"))
```

Comment:
What about Windows 32-bit?

Reply:
Yeah, will add 32-bit support

Reply:
Actually, I don't think any change is needed here.  I believe we build x86 running as x64, so we can still use the x64 rcedit to modify the x86 binary.

Reply:
Clear. Thanks.

Closed.